modules/subchannel/src/problems/QuadSubChannel1PhaseProblem.C

```diff
@@ -356,220 +356,70 @@ QuadSubChannel1PhaseProblem::computeFrictionFactor(FrictionStruct friction_args)
   }
 }
 
-void
-QuadSubChannel1PhaseProblem::computeWijPrime(int iblock)
+Real
+QuadSubChannel1PhaseProblem::computeBeta(int i_gap, int iz)
```

Comment:
These should all be `unsigned int` too. Not need to change now really, but for future reference to save memory where we can